### PR TITLE
Che 756: make the loading view fit the height of the screen to avoid scrolling

### PR DESCRIPTION
**Pull request by akurinnoy**
**_1 Upvote**_

### CODE REVIEW

**Comment by ashumilova** [comment]
ok

**Comment by olexii4** [comment]
+1

File: dashboard/src/components/widget/loader/che-loader-crane.directive.js [hunk 1]
```diff
@@ -77,6 +79,42 @@ export class CheLoaderCrane {
...
+
+      // initial resize
+      setCraneSize();
+
+      destroyResizeEvent = angular.element(this.$window).bind('resize', (event) => {
+        console.log('>>> window resize stopped with event: ', event);
+
+        setCraneSize();
+
+        // re-init all animations
```
Comment (ashumilova) [inline dashboard/src/components/widget/loader/che-loader-crane.directive.js]: remove if not needed

File: dashboard/src/app/ide/ide-loader/ide-loader.html [hunk 2]
```diff
@@ -1,16 +1,19 @@
 <div class="ide-loader" layout="column" layout-align="center center">
   <che-loader layout="column" layout-align="center center">
     <!--crane and terminals-->
-    <div class="che-loader-animation-panel main-page" layout="row">
-      <div flex="initial" flex-gt-sm="60">
+    <div class="che-loader-animation-panel main-page" layout="row" flex="">
```
Comment (ashumilova) [inline dashboard/src/app/ide/ide-loader/ide-loader.html]: flex="" can be just flex

File: dashboard/src/components/widget/loader/che-loader-crane.directive.js [hunk 3]
```diff
@@ -20,10 +20,10 @@ export class CheLoaderCrane {
    * Default constructor that is using resource
    * @ngInject for Dependency injection
    */
-  constructor($timeout) {
+  constructor($timeout, $window) {
     this.$timeout = $timeout;
+    this.$window = $window;
     this.restrict = 'E';
-    this.replace = true;
```
Comment (benoitf) [inline dashboard/src/components/widget/loader/che-loader-crane.directive.js:26]: this removal is not linked to the description of the commit

**Comment by ashumilova** [comment]
please, give a look at these notifications:

ERROR in ./src/components/widget/loader/che-loader-crane.directive.js

/home/Ann/codenvy/che/dashboard/src/components/widget/loader/che-loader-crane.directive.js
   96:13  error  You should use the angular.isFunction method            angular/typecheck-function
  108:74  error  "event" is defined but never used                       no-unused-vars
  118:10  error  Redundant double negation in an if statement condition  no-extra-boolean-cast
  133:36  error  "jQuery" is not defined                                 no-undef
  141:27  error  "jQuery" is not defined                                 no-undef
  150:27  error  "jQuery" is not defined                                 no-undef

✖ 6 problems (6 errors, 0 warnings)

**Comment by ashumilova** [comment]
resizing the crane seems ok

**Comment by akurinnoy** [comment]
fixed

**Comment by ashumilova** [comment]
squash 1st and last commit and merge please

**Pull request merged**